src/python/pants/engine/internals/selectors.py

```diff
@@ -79,6 +79,10 @@ def __str__(self) -> str:
 
 
 class Call(PyGeneratorResponseCall):
```

Comment:
Should these be in the parent in `native_engine.pyi` similar to how the Get call works? As in, satisfying the type checker in this file?

Though, there is a weird TODO below it for the Getters, so I'm not sure:

```
# TODO: Conditional needed until Python 3.8 allows the subscripted type to be used directly.
```

Reply:
Probably a good idea!

As for the comment, it is referring to the fact that `PyGeneratorResponseGet` is a  generic _stub_ with subscript which doesn't work under 3.8.

Further, there's no type information of `Call` that can be checked, so the output type hint is not relevant, at least not for this change.

Reply:
Oh, yeah, sorry, I just meant - how old was that code that we still have the TODO 😆 

My assumption is that, if that's still there, it might be there with a reason as this file's code has been modded several times